pontoon/base/static/js/translate.js

```diff
@@ -728,6 +742,12 @@ var Pontoon = (function (my) {
 
       self.fluent.toggleButton();
 
+      // Toggle read-only mode
+      $('#editor #single').toggleClass('readonly', entity.readonly);
+      $('#editor textarea').attr('readonly', function() {
+        return self.isReadonlyEditor();
+      });
```

Comment:
Because when all projects are loaded for a locale (e.g. https://pontoon.mozilla.org/sl/all-projects/all-resources/), some strings could be read-only, and some not, so we need to make a check on each entity switch.